test/tests/dgkernels/adaptivity/tests

```diff
@@ -0,0 +1,14 @@
+[Tests]
```

Comment:
These two tests need to use the new `requirements` syntax to name the requirement and link back to the issue.